Tasks/DownloadFileShareArtifactsV0/task.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "E3CF3806-AD30-4EC4-8F1E-8ECD98771AA0",
+  "name": "DownloadFileShareArtifacts",
+  "friendlyName": "Download FileShare Artifacts",
+  "description": "Download FileShare Artifacts",
+  "helpMarkDown": "",
+  "category": "Utility",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 136,
+    "Patch": 0
+  },
+  "groups": [
+    {
+        "name": "advanced",
+        "displayName": "Advanced",
+        "isExpanded": false
+    }
+  ],
+  "demands": [],
+  "inputs": [
+    {
+      "name": "location",
+      "type": "string",
+      "label": "Artifact location",
+      "required": true,
+      "helpMarkDown": "Fileshare location"
+    },
+    {
+      "name": "artifactName",
+      "type": "string",
+      "label": "Artifact name",
```

Comment:
In the inputs we should drop "artifact", is this the name of the folder?

Reply:
This should be the name using which the artifact was published in build.